lib/features/email/data/repository/calendar_event_repository_impl.dart

```diff
@@ -49,4 +53,24 @@ class CalendarEventRepositoryImpl extends CalendarEventRepository {
     return _calendarEventDataSource[DataSourceType.network]!
       .rejectEventInvitation(accountId, blobIds, language);
   }
+
+  @override
+  Future<List<BlobCalendarEvent>> transformCalendarEventDescription(
+    List<BlobCalendarEvent> blobCalendarEvents,
+    TransformConfiguration transformConfiguration,
+  ) async {
+    return Future.wait(blobCalendarEvents.map((blobCalendarEvent) async {
+      return BlobCalendarEvent(
+        blobId: blobCalendarEvent.blobId,
+        calendarEventList: await Future.wait(blobCalendarEvent.calendarEventList.map((calendarEvent) async {
+          return calendarEvent.copyWith(
+            description: await _htmlDataSource.transformHtmlEmailContent(
```

Comment:
Please explicitly verify that this HTML display is not affected by XSS and would drop advance XSS scenario like ondisplay calls.

Reply:
- Xss test already available at https://github.com/linagora/tmail-flutter/blob/37b83e19d6c9dcad7ee26d3b71ef153c4a82e9f7/core/test/utils/standardize_html_sanitizing_transformers_test.dart
- This PR is also updated to reflect xss preventation